From 5c98f501eb9e63320dccc111b8926ebc40a16a9e Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:31 -0400
Subject: [redhat] kabi: Adding symbol __root_device_register (drivers/base/core.c)

Message-id: <5ecac0f34836e818c4fe8eeaa75baebf90956018.1412771859.git.poros@redhat.com>
Patchwork-id: 97160
O-Subject: [RHEL7.1 PATCH 11/67] kabi: Adding symbol __root_device_register (drivers/base/core.c)
Bugzilla: 1148035
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148035
    https://bugzilla.redhat.com/show_bug.cgi?id=1148035

    Description:
    ============
    This is used with low level character devices created
    by the extint.ko, ioc4_extint.ko, and pcie_rt.ko modules. device_unregister is
    already whitelisted.
    Symbol used in hwperf.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__root_device_register
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__root_device_register
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__root_device_register

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__root_device_register b/redhat/kabi/kabi-module/kabi_ppc64/__root_device_register
new file mode 100644
index 0000000..0533b4a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__root_device_register
@@ -0,0 +1,2 @@
+#1-
+0xafdf7cb7 __root_device_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__root_device_register b/redhat/kabi/kabi-module/kabi_s390x/__root_device_register
new file mode 100644
index 0000000..2bee6b4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__root_device_register
@@ -0,0 +1,2 @@
+#1-
+0xba909aa0 __root_device_register vmlinux EXPORT_SYMBOL_GPL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__root_device_register b/redhat/kabi/kabi-module/kabi_x86_64/__root_device_register
new file mode 100644
index 0000000..f2d6a21
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__root_device_register
@@ -0,0 +1,2 @@
+#1-
+0x934ffc58 __root_device_register vmlinux EXPORT_SYMBOL_GPL
-- 
1.7.1